cypress/support/step_definitions/manualAccountCreation/manualAccountCreation.ts

```diff
@@ -334,8 +334,8 @@ When('I enter incorrect home telephone number {string}', (incorrectHomeTelephone
   contactDetails.enterHomeTelephone(incorrectHomeTelephone);
 });
 
-When('I enter incorrect business telephone number {string}', (incorrectBusinessTelephone: string) => {
-  contactDetails.enterBusinessTelephone(incorrectBusinessTelephone);
+When('I enter incorrect work telephone number {string}', (incorrectWorkTelephone: string) => {
+  contactDetails.enterWorkTelephone(incorrectWorkTelephone);
```

Comment:
generic steps can be used instead

Reply:
This has to be deleted, never used in the tests